lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
```

Comment:
I'd rather see `value.getTime()` and compare that to an invalid date's `getTime()` that way it's obvious and no comment is required.

```js
const invalidDate = Date.parse("invalid-date-string");
...
// later on
 if (isDate(value)) {
    if(Object.is(value.getTime(), invalidDate.getTime()) {
    //...
```